react/week1/todos/src/index.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+// import App from './App';
+// import * as serviceWorker from './serviceWorker';
+
+
+const root = document.getElementById('root')
+
+ function TodoItem(props) {
+   return <li>{props.description}, {props.deadline}</li>
+ }
+
+class Todo extends React.Component {
+  render() {
+   const todoList = this.props.todos;
+   const result = todoList.map(element => {
+    return <TodoItem description={element.description} deadline={element.deadline}></TodoItem>
+  })
+   return (
+     <ul>{result}</ul>
+   )
+  }
+}
+
+const todosArray = [
+  { description: 'Watch TV', deadline: new Date('2020-04-24').toDateString()},
+  { description: 'Listen music', deadline: new Date('2020-02-20').toDateString()},
+  { description: 'Wash clothes', deadline: new Date('2020-03-28').toDateString()}
+]
+
+class Header extends React.Component {
+  render() {
+    return (
+    <header>
+      <h3>Todo list</h3>
+    </header>)
+  }
+}
+
+class Footer extends React.Component {
+  render() {
+    return (
+      <footer>
+        <h4>Have a nice day!</h4>
+      </footer>
+    )
+  }
+}
+
+ReactDOM.render(
+  <div id="todo">
+    <Header></Header>
+    <Todo todos={todosArray}></Todo>
+    <Footer></Footer>
```

Comment:
When you're not passing anything inside the component tags you can just write: 
```suggestion
    <Header />
    <Todo todos={todosArray} />
    <Footer />
```